layouts/partials/head.html

```diff
@@ -133,6 +133,9 @@
     <!-- Preload the component JavaScript to help our Google Page Speed score. -->
     <link rel="preload" href="/js/components/components.esm.js" as="script" crossorigin="anonymous">
 
+    <!-- Google Site Verification for Google Webmaster tools and YouTube. -->
+    <meta name="google-site-verification" content="N-ezSTIu4P3bSc4TqidV4wWCkMzFiMN269ZgDYArGkk" />
```

Comment:
There's already a `<meta name="google-site-verification"` on line 5. Do we not need that one anymore? Can we put this one just below it?

Reply:
I ended up just replacing the old one with the new one. This shouldn't break anything but I'll be monitoring our linked Google accounts and will re-verify if anything falls out of sync.